README.MD

```diff
@@ -2,61 +2,47 @@
 
 # Project Name
 
-> Description the project.
+> Capstone project created using [this](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design template.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Javascript
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+You can view the deployed project [here](https://abbynyakara.github.io/Capstone-1/). The presentation for the project can also be viewed [here] (https://www.loom.com/share/2a0c91ad68a94e2faa727d450d35b1e2)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+- The project was originally designed by [Candance-Chin]
```

Comment:
- [ ] Kindly replace this with "The credit of design of this website goes to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/)."